docs/src/operations/requirements.md

```diff
@@ -18,12 +18,19 @@ transaction for each block the validator agrees with, which can cost up to
 
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
+**NOTE**: These recommendations may be out of date as the network grows and resource requirements change.
+
```

Comment:
I would leave this out and just make sure to keep this updated.

Reply:
Removed in abfdac7076